app/views/registrations/__new.html.erb

```diff
@@ -1,18 +1,6 @@
-<!DOCTYPE html>
```

Comment:
I left this file that you created in the app for reference, but renamed it so it wouldn't get in the way.